src/localization/en.json

```
@@ -4,6 +4,7 @@
     "cancel": "Cancel",
     "edit": "Edit",
     "edited": "Edited",
+    "ok": "OK",
```

Do not use uppercase
```suggestion
    "ok": "Ok",
```